src/components/Notification/Notification.tsx

```diff
@@ -0,0 +1,175 @@
+import classNames from "classnames";
+import PropTypes from "prop-types";
+import React, { useEffect, useRef } from "react";
+import type { HTMLProps, ReactNode } from "react";
+
+import Button, { ButtonAppearance } from "../Button";
+import type { Props as ButtonProps } from "../Button";
+
+export const notificationTypes = {
```

Comment:
It's repeated in the TS Props definition. Maybe this can be a TS enum?

Reply:
Replaced with `NotificationSeverity` and used in TS Props and PropTypes